ARCHITECTURE.md

```diff
@@ -32,7 +32,7 @@ This is the docs website code. It comes from the dbt-docs repository, and is gen
 ## Adapters
 
 dbt uses an adapter-plugin pattern to extend support to different databases, warehouses, query engines, etc. 
-Note: dbt-postgres used to exist in dbt-core but is now in [a separate repo](https://github.com/dbt-labs/dbt-adapters/dbt-postgres) 
+Note: dbt-postgres used to exist in dbt-core but is now in [a dbt-adapters repo](https://github.com/dbt-labs/dbt-adapters/tree/main/dbt-postgres) 
```

Comment:
```suggestion
Note: dbt-postgres used to exist in dbt-core but is now in [the dbt-adapters repo](https://github.com/dbt-labs/dbt-adapters/tree/main/dbt-postgres) 
```

We should probably switch the indefinite article "a" to the definite article "the" to satisfy the pedants out there (myself included) 😂